CHANGELOG.md

```diff
@@ -1,3 +1,23 @@
+CHANGES IN VERSION 1.9.1 (from 1.9.0)
+===================================
+
+Appium 1.9.1 is a patch release
+
+#### iOS
+* Support Xcode 10 and iOS 12.0
+* Fix container type parsing
```

Comment:
The more important change is https://github.com/appium/appium-xcuitest-driver/commit/5e1ae26644dbb060c3b8594b2f4ac6bea03bc7fa